packages/jest-schemas/src/index.ts

```diff
@@ -10,6 +10,7 @@ import {Static, Type} from '@sinclair/typebox';
 const RawSnapshotFormat = Type.Partial(
   Type.Object({
     callToJSON: Type.Readonly(Type.Boolean()),
+    compareKeys: Type.Readonly(Type.Null()),
```

Comment:
this looks somewhat weird but from what I can tell this `RawSnapshotFormat` is supposed to declare type accepted by the `jest-config` so with that in mind I think this is correct (based on other options that are declared here)

Reply:
Yep, correct 👍